### PR TITLE
Feature/improve sync timeout

### LINKED ISSUE #26: Improve synchronous processing #1
We should investigate the 5MB limitation and find a different solution that will enable running of synchronous processes for larger areas. Currently only very small areas are supported. 

First goal could be to support running of NDVI for 1 date for 1 MGRS tile (~100 km *100 km).

### LINKED ISSUE #63: move /.well-known/openeo
Currently exposed under: https://openeo.sentinel-hub.com/production/.well-known/openeo but should not be part of the main API tree and not versioned (can be a static file) --> should be moved to https://openeo.sentinel-hub.com/.well-known/openeo

Not sure if it's possible (to be checked with Primož) but https://sentinel-hub.com/.well-known/openeo would be even better.

### PR DESCRIPTION
**Pull request by guss84**
closes #26 
closes #63 

- The server can now be deployed on EC2, where we can increase the timeout
- Removed zappa as we are not using Lambda anymore
- Removed check that limits file size, as the limiting factor is pixel dimensions of 2500X2500(from Sentinelhub api)
- Added [gunicorn](https://gunicorn.org/) for our production WSGI server
- The .well-known endpoint is still in the same location, but in production we can alter the location of the endpoint through nginx so that https://openeo.sentinel-hub.com/.well-known/openeo listens to  https://openeo.sentinel-hub.com/production/.well-known/openeo

### CODE REVIEW

File: rest/processing/process.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ class Process:
     def __init__(self, process, width=None, height=None, user=User(), user_defined_processes={}, request_type=None):
         self.DEFAULT_EPSG_CODE = 4326
         self.DEFAULT_RESOLUTION = (10, 10)
-        self.MAXIMUM_SYNC_FILESIZE_BYTES = 5000000
+        self.MAXIMUM_SYNC_FILESIZE_BYTES = 100000000
```
Comment (zansinergise) [inline rest/processing/process.py]: This is not used anywhere anymore I believe, so it can probably be removed

**Review by zansinergise — CHANGES_REQUESTED** [review]
Otherwise looks good

File: .github/workflows/deploy_production.yaml [hunk 2]
```diff
@@ -34,7 +36,7 @@ jobs:
         run: |
           mkdir -p ~/.aws
           echo -e "[default]\nregion=eu-central-1" > ~/.aws/config
-          echo -e "[default]\naws_access_key_id=${{ secrets.PRODZAPPA_AWS_ACCESS_KEY_ID }}\naws_secret_access_key=${{ secrets.PRODZAPPA_AWS_SECRET_ACCESS_KEY }}" > ~/.aws/credentials
```
Comment (zcernigoj) [inline .github/workflows/deploy_production.yaml:37]: After this is merged, we can remove the [secrets](https://github.com/Open-EO/openeo-sentinelhub-python-driver/settings/secrets/actions) that were used for deployment with Zappa

File: rest/processing/process.py (hunk 1 above)
Comment (guss84) [reply rest/processing/process.py]: Removed

File: .github/workflows/deploy_production.yaml (hunk 2 above)
Comment (guss84) [reply .github/workflows/deploy_production.yaml:37]: Agreed, we can also remove it once we delete the lambda functions.

**Review by zansinergise — APPROVED** [review]

**Pull request merged**